<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_8">
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_13">
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item row="0" column="1">
         <widget class="QPushButton" name="button_banljus_off">
          <property name="minimumSize">
           <size>
            <width>1</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>av</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Banljus</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="button_banljus_on">
          <property name="minimumSize">
           <size>
            <width>1</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>på</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="tab_j">
         <attribute name="title">
          <string>J</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <widget class="QPushButton" name="button_j_reload">
            <property name="text">
             <string>Fyll på vapen</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="button_fire">
            <property name="text">
             <string>Avfyra</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="button_j_target1">
            <property name="text">
             <string>Lås mål 1</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Vapenval ST</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <property name="spacing">
                  <number>6</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="button_j_st_hvp">
                   <property name="text">
                    <string>HVP (Radar RB99)</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="button_j_st_ir">
                   <property name="text">
                    <string>IR (RB74)</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="button_j_st_akan">
                   <property name="text">
                    <string>AKAN</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="button_j_targetflight">
            <property name="text">
             <string>Flytta mål</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="button_j_mod">
            <property name="text">
             <string>Välj huvudmod Jakt</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Vapenväljaren</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="1" column="2">
                <widget class="QPushButton" name="button_j_sijak">
                 <property name="minimumSize">
                  <size>
                   <width>1</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>SIJAK</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="button_j_spak_ak">
                 <property name="minimumSize">
                  <size>
                   <width>1</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>AK</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="button_j_spak_hvp">
                 <property name="minimumSize">
                  <size>
                   <width>1</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>HVP</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="button_j_sijir">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>1</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="baseSize">
                  <size>
                   <width>1</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>SIJIR</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_a">
         <attribute name="title">
          <string>A</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="button_a_mod">
              <property name="text">
               <string>Välj huvudmod Attack</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_a_fire">
              <property name="text">
               <string>Avfyra</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_a_reload">
              <property name="text">
               <string>Fyll på vapen</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Vapenval ST</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="spacing">
                  <number>6</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="button_a_st_hvp">
                   <property name="text">
                    <string>HVP (Raketer)</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="button_a_st_ir">
                   <property name="text">
                    <string>IR (IR robot)</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="button_a_st_akan">
                   <property name="text">
                    <string>AKAN</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Vapenväljaren</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_7">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item row="1" column="2">
                  <widget class="QPushButton" name="button_a_sijak">
                   <property name="minimumSize">
                    <size>
                     <width>1</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>SIJAK</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QPushButton" name="button_a_spak_ak">
                   <property name="minimumSize">
                    <size>
                     <width>1</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>AK</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QPushButton" name="button_a_spak_hvp">
                   <property name="minimumSize">
                    <size>
                     <width>1</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>HVP</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QPushButton" name="button_a_sijir">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>1</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="baseSize">
                    <size>
                     <width>1</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>SIJIR</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_s">
         <attribute name="title">
          <string>S</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QPushButton" name="button_s_mod">
              <property name="text">
               <string>Välj huvudmod Spaning</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_land">
         <attribute name="title">
          <string>Land</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_11">
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item row="3" column="0">
             <widget class="QGroupBox" name="groupBox_7">
              <property name="title">
               <string>Flygplats</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_18">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_17">
                 <property name="bottomMargin">
                  <number>10</number>
                 </property>
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item row="1" column="0">
                  <widget class="QComboBox" name="comboBox_airports">
                   <property name="editable">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <layout class="QGridLayout" name="gridLayout_16">
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item row="0" column="1">
                    <widget class="QPushButton" name="land_bi">
                     <property name="text">
                      <string>BI-B</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QPushButton" name="land_hb">
                     <property name="text">
                      <string>HB</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="0" column="0">
                  <widget class="QPushButton" name="button_land_update">
                   <property name="text">
                    <string>Uppdatera Lista</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QGroupBox" name="groupBox_6">
                   <property name="title">
                    <string>Riktning</string>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                     <widget class="QPushButton" name="land_rikt_n">
                      <property name="text">
                       <string>normal</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="land_rikt_inv">
                      <property name="text">
                       <string>omvänd</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>Snabbval</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_15">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="horizontalSpacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_14">
                 <item row="0" column="0">
                  <widget class="QPushButton" name="snabb_eskn_1">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>ESKN 26</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QPushButton" name="snabb_eskn_3">
                   <property name="minimumSize">
                    <size>
                     <width>10</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>36</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QPushButton" name="snabb_eskn_2">
                   <property name="minimumSize">
                    <size>
                     <width>10</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>08</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QPushButton" name="snabb_eskn_4">
                   <property name="minimumSize">
                    <size>
                     <width>10</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>16</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>TILS</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="land_set_all">
              <property name="text">
               <string>Alla Mod till Landning</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="button_land_mod">
              <property name="text">
               <string>Välj huvudmod Landning</string>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string>Tanka</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="button_tanka_20">
             <property name="text">
              <string>20 (larm)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_tanka_50">
             <property name="text">
              <string>50</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_tanka_100">
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>216</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
